bin/node/src/aleph_node_rpc.rs

```diff
@@ -50,64 +75,92 @@ impl From<Error> for JsonRpseeError {
                 e,
                 None::<()>,
             )),
+            Error::BlockWithoutDigest => CallError::Custom(ErrorObject::owned(
+                BLOCK_WITHOUT_DIGEST_ERROR,
+                "Block doesn't have any Aura pre-runtime digest item.",
+                None::<()>,
+            )),
+            Error::SessionInfoNotAvailable => CallError::Custom(ErrorObject::owned(
+                SESSION_INFO_NOT_AVAILABLE_ERROR,
+                "Failed to get session data at the parent block.",
+                None::<()>,
+            )),
+            Error::AuthoritiesInfoNotAvailable => CallError::Custom(ErrorObject::owned(
+                AUTHORITIES_INFO_NOT_AVAILABLE_ERROR,
+                "Failed to get authority set at the parent block.",
+                None::<()>,
+            )),
         }
         .into()
     }
 }
 
 /// Aleph Node RPC API
 #[rpc(client, server)]
-pub trait AlephNodeApi<Hash, Number> {
+pub trait AlephNodeApi<Block: BlockT> {
```

Comment:
I wasn't aware of that... while I could make this trait dependent on `Hash` and `Number` generic parameters, but I have to have `Block` parameter somehow constrained (for `ProvideRuntimeApi<Block>`) - either in `AlephNodeApi` or in `AlephNode`

Reply:
We are slowly moving to having more of the (near) top level things not parametrized at all, and just using type( aliase)s directly, including a specific block type. It would be great if this could still only use `Hash` and `Number`, but there will likely be a bigger refactor of all this anyway, so if it's really hard to achieve (I didn't look hard enough to fully understand what the problem is, but definitely not in the API itself) you can leave it as-is.